test_unstructured_ingest/expected-structured-output/pdf-fast-reprocess/azure/IRS-form-1987.pdf.json

```diff
@@ -57,10 +37,10 @@
       ],
       "page_number": 1
     },
-    "text": "Internal Revenue Service"
+    "text": "Department of the Treasury Internal Revenue Service"
```

Comment:
Yes, I combined those two lines because I thought it looked better (referring to the result JSON for the [`hi_res` strategy](https://github.com/Unstructured-IO/unstructured/blob/main/test_unstructured_ingest/expected-structured-output/azure/IRS-form-1987.pdf.json#L19)). What do you think?

Reply:
Yeah combine the two makes sense here. Also I see the ingest combine words together to get one line like `Sections B-2 and B-3`, which is definitely better! Only one place I am not sure is for `"Specific Instructions Section A"` in the changes below, I feel like separate them into `"Specific Instructions"` and  `"Section A"` would be better.